runtime/common/src/paras_registrar.rs

```diff
@@ -584,6 +586,7 @@ impl<T: Config> Pallet<T> {
 		let res2 = runtime_parachains::schedule_parathread_upgrade::<T>(to_upgrade);
 		debug_assert!(res2.is_ok());
 		T::OnSwap::on_swap(to_upgrade, to_downgrade);
```

Comment:
What is the reason we switched the order? The input params are:
```rust
fn do_thread_and_chain_swap(to_downgrade: ParaId, to_upgrade: ParaId)
```
but the params are switched here:
```rust
T::OnSwap::on_swap(to_upgrade, to_downgrade);
```
Why not keep it the same.

Reply:
Looks like a bug?

Reply:
I do not believe it was a bug, just a way to reuse `do_thread_and_chain_swap()` for parathread and parachain and vice versa